app/retail/templates/landing/contributor/explorer.html

```diff
@@ -0,0 +1,32 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static cookielaw_tags %}
+<link rel="stylesheet" href={% static "v2/css/dashboard.css" %}>
+<link rel="stylesheet" href={% static "v2/css/profile.css" %}>
+<link rel="stylesheet" href={% static "v2/css/tag.css" %}>
+<div>
+  <div class="col text-center">
+    <!-- TODO : Language Logo  -->
```

Comment:
i think this only needs to be displayed if its a programming language for which we have a logo.  if not, it should gracefully degrade